Assets/Materials/ReferenceImage.mat

```diff
@@ -2,14 +2,18 @@
 %TAG !u! tag:unity3d.com,2011:
 --- !u!21 &2100000
 Material:
-  serializedVersion: 6
+  serializedVersion: 8
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: ReferenceImage
   m_Shader: {fileID: 4800000, guid: 8c9bb7751e2a42747aa2a55e4108bde8, type: 3}
-  m_ShaderKeywords: _ALPHATEST_ON _LIGHTMAPPING_DYNAMIC_LIGHTMAPS _UVSEC_UV1
+  m_ValidKeywords: []
```

Comment:
was this change intended?

Reply:
No. I tend to assume Unity is doing the right thing when it modifies materials in response to changes I made in the UI - but this oe does look a little sus.

Reply:
I think we're going to have to trust Unity on this one - that setting doesn't correlate with anything in the UI - or with anything I changed in the shader - so it might just be an old property that Unity is pruning.

Reply:
ok